site/content/en/docs/start/_index.md

```diff
@@ -47,10 +47,23 @@ curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-latest
 sudo rpm -ivh minikube-latest.x86_64.rpm
 ```
 
+### Homebrew package
+
+```shell
+brew install minikube
+```
+
+If `which minikube` fails after installation via brew, you may have to remove the minikube cask and link the binary:
```

Comment:
I rather we keep this first page short, about creating a full page for all the possible installation for linux, and then from here

that says
for hombrew and full notes see ...(link)